x-pack/test/functional/apps/security/doc_level_security_roles.ts

```diff
@@ -20,6 +20,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const kibanaServer = getService('kibanaServer');
 
   describe('dls', function () {
+    const customUserName = 'userEast';
```

Comment:
Question: These changes are definitely cleaner, were they related to the CodeEditor or came from a merge? Just wondering!

Reply:
They are somewhat related to the code editor changes, I had cause to review this test file whilst trying to get this particular test to pass with the update to the code editor, so I made the change. The actual change that was required is [here](https://github.com/elastic/kibana/pull/171720/files/bbd8a254d315ef668916c4f28fa18e02fbf411a6#diff-51114839ee8f602bbcb0ff42f5bb0255290f37fe9580aeba9e730bca286e7f11)